descwl_shear_sims/galaxies.py

```diff
@@ -218,7 +219,7 @@ def get_objlist(self, *, survey):
             objlist.append(self._get_galaxy(flux))
             shifts.append(galsim.PositionD(sarray['dx'][i], sarray['dy'][i]))
 
-        return objlist, shifts, redshifts
+        return objlist, shifts, redshifts, indexes
```

Comment:
This is a breaking change.

Also the returned data is getting to be a lot.  Would we add more in the future?  Probably.

Maybe we should make this return a dict `{'objlist': objlist, 'shifts': shifts, ...}`

Since we are already making a breaking change this would be OK.

We should have a galaxies unit test that works with these and makes sure everything is constent.

Reply:
what do you think about this?